### ISSUE TITLE
two table has many n:m associate how to select?

### ISSUE THREAD

**Issue by callmesoul** [sequelize/sequelize#9298]
has two table User and Bucket;

their associate:

```
models.User.belongsToMany(models.Bucket,{through:"user_bucket"});
models.User.belongsToMany(models.Bucket,{as:"hasUploadUser",through:"user_bucket_upload"});

models.Bucket.belongsToMany(models.User,{through:"user_bucket"});
models.Bucket.belongsToMany(models.User,{as:"hasUploadUser",through:"user_bucket_upload"});
```

now  how i select associate hasUploadUser?

```
model.Bucket.findAll({
            include:[
                {
                    model:model.User,
                    as:"hasUploadUser",
                }
            ]
})
```
this select result is this asscsiate `models.User.belongsToMany(models.Bucket,{through:"user_bucket"});`

**Comment by sushantdhiman** [comment]
Can you please explain your issue in simpler terms, what are you trying to do :)

------------------------
Please use Github Issue Tracker only for reporting bugs, requesting new features or discussions. Ask questions on [Stackoverflow sequelize.js tag](https://stackoverflow.com/questions/tagged/sequelize.js) or [Slack](sequelize.slack.com). While I would like to help answer any questions but it still take too much time.

If you are reporting a bug please isolate it as SSCCE, present it nicely and follow issue template. This will help maintainers and contributors understand your issue quickly.

If you are requesting a feature spend some time to properly present your use case with some examples, current and expected outcome.

( [What is SSCCE ?](http://sscce.org) You can find template for Sequelize [here](https://github.com/sequelize/sequelize/blob/master/sscce_template.js) )

A few good example of SSCCE

1) https://github.com/sequelize/sequelize/issues/8014#issue-245907597
2) https://github.com/sequelize/sequelize/issues/7318#issuecomment-284136530
3) https://github.com/sequelize/sequelize/issues/8749#issuecomment-354734396
